README.md

```diff
@@ -1,22 +1,37 @@
-# Turborepo starter
+# EventSea: A Decentralized Events Platform
 
-This is an official starter Turborepo.
+---
 
-## Using this example
+### Overview:
 
-Run the following command:
+EventSea is a decentralized application (DApp) designed to revolutionize the way events are organized and attended. Built on the top of Linea, EventSea offers a transparent, secure, and user-centric platform for event creation and ticketing.
```

Comment:
what if we omit words such as "revolutionise" for now. We may be in the future :)
but not now